eng/scripts/Language-Settings.ps1

```diff
@@ -161,68 +161,200 @@ function Get-python-GithubIoDocIndex()
   GenerateDocfxTocContent -tocContent $tocContent -lang "Python" -campaignId "UA-62780441-36"
 }
 
-# Updates a python CI configuration json.
-# For "latest", the version attribute is cleared, as default behavior is to pull latest "non-preview".
-# For "preview", we update to >= the target releasing package version.
-function Update-python-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$null)
-{
-  $pkgJsonLoc = (Join-Path -Path $ciRepo -ChildPath $locationInDocRepo)
+function ValidatePackage($packageName, $packageVersion, $workingDirectory) {
+  $packageExpression = "$packageName$packageVersion"
+  Write-Host "Validating $packageExpression"
+
+  $installTargetFolder = Join-Path $workingDirectory $packageName
+  New-Item -ItemType Directory -Force -Path $installTargetFolder | Out-Null
+
+  # TODO: Additional checks for package validation in docs build.
```

Comment:
Is this a real TODO with a specific set of things you want to add? If so please file an issue to track it, if not probably remove the TODO word.

Reply:
I think we should replicate as much of the docs CI process as we possibly can for each package that we want to test. It'll give us the greatest confidence that the docs CI process will build docs artifacts.